.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  release:
+    types:
+      - created
```

Comment:
Note that this is triggered by the creation of a "release".

This means you still follow https://github.com/chanzuckerberg/axe-storybook-testing/pull/75 as normal and publish to npm.

When the tags get pushed as part of that process, this job will be triggered and also publish to github.

(Hat tip @andremalan for developing this concept)